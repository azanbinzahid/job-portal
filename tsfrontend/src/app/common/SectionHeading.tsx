import React, { FC } from "react";
import { Container } from "react-bootstrap";
import styled from "styled-components";

const Header = styled.div`
  h1 {
    font-family: "Jost", "PT Serif", sans-serif;
    font-weight: 700;
    font-size: 6vmin;
  }
  h3 {
    font-family: "Jost", "PT Serif", sans-serif;
    font-weight: 300;
    font-size: 3vmin;
  }
`;

type Props = {
  title: string;
  subtitle?: string;
};

const SectionHeading: FC<Props> = ({ title, subtitle }) => {
  return (
    <Container<React.ElementType> fluid align="center" className="pb-5 pt-5">
      <Header>
        <h1> {title} </h1>
        <h3> {subtitle} </h3>
      </Header>
    </Container>
  );
};

export default SectionHeading;
