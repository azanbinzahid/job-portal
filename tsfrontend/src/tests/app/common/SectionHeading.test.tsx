import React from "react";
import { shallow } from "enzyme";
import SectionHeading from "app/common/SectionHeading";
test("renders the component", () => {
  const component = shallow(<SectionHeading title="abc" />);
  expect(component).toMatchSnapshot();
});
