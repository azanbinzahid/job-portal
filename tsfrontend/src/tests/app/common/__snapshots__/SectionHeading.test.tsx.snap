// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<Container
  align="center"
  className="pb-5 pt-5"
  fluid={true}
>
  <styled.div>
    <h1>
       
      abc
       
    </h1>
    <h3>
       
       
    </h3>
  </styled.div>
</Container>
`;
